### PR TITLE
Fix PE-D bugs assigned to Hieu

### PR DESCRIPTION
**Pull request by hieunm1821**

### CODE REVIEW

**Review by xzzz3 — APPROVED** [review]
just saw this -- looks good!

**Review by zxgoh — APPROVED** [review]
LGTM!

File: src/main/java/seedu/address/model/driver/NameDriver.java [hunk 1]
```diff
@@ -48,7 +48,7 @@ public String toString() {
     public boolean equals(Object other) {
         return other == this // short circuit if same object
                 || (other instanceof NameDriver // instanceof handles nulls
-                && fullName.equals(((NameDriver) other).fullName)); // state check
+                && fullName.toLowerCase().equals(((NameDriver) other).fullName.toLowerCase())); // state check
```
Comment (jaegarpoon) [inline src/main/java/seedu/address/model/driver/NameDriver.java:51]: Can use equalsignorecase for future implementations if it helps! But same purpose :)

**Review by jaegarpoon — APPROVED** [review]
Looks good!

**Pull request merged**